vxquery-core/src/main/java/org/apache/vxquery/xmlparser/SAXContentHandler.java

```diff
@@ -66,6 +66,7 @@
     protected final List<ElementNodeBuilder> enbStack;
     protected final List<ElementNodeBuilder> freeENBList;
     protected boolean isIndexHandler;
+    protected boolean index;
```

Comment:
I think it should be possible to use this class for indexing with out making any indexing specific changes. I think the changes are here because the class is not being used as its intended. I don't think indexing is doing anything that requires special consideration.